### ISSUE TITLE
NTR:  Increased echogenicity of the renal parenchyma 

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4658]
**Preferred term label:**
 Increased echogenicity of the renal parenchyma 

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increase in the echogenicity of the renal parenchyma; this generally refers to how bright or dark the kidney parenchyma appears in comparison to the liver.
Source: PMID:24235286

**Parent term (use hpo.jax.org/app)**
HP_0012211 'Abnormal renal physiology'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/616809?search=616809&highlight=616809
(I am reviewing the annotations for diseases mentioned in the GPI paper)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by pnrobinson** [comment]
Comment: 
![image](https://user-images.githubusercontent.com/4943420/55655775-15f30c80-57c3-11e9-8436-58f8ec154722.png)
PMID: 2259758

**Comment by pnrobinson** [comment]
We have this: https://hpo.jax.org/app/browse/term/HP:0004719

I think it is the same, but let's take a look if we need to differentiate hyperechogenicity of parts of the kidney

**Comment by pnrobinson** [comment]
Yes we probably do want a mini hierarchy here. This is something we can discuss at the kidney workshop. I will postpone action on this for now, please leave this term open.

**Comment by pnrobinson** [comment]
@avizrosenberg
@yongqunh  
@beckysteck

This was a term request outside of the KPMP workshop. I think that we could probably use a small subontology that would reflect PMID: 2259758. Is this a good idea?

**Comment by avizrosenberg** [comment]
This is a feature at the US and not histologic level

**Comment by pnrobinson** [comment]
After looking at this, I think we should retain just the one term and add the synonym (done).
